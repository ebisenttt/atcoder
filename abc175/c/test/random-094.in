-72356955269571 711255676227271 651216710919797
