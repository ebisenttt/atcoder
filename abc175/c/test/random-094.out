578859755650226
